### DISCUSSION TITLE
鼠标悬浮到节点文字显示提示信息不起作用

### DISCUSSION THREAD

**Discussion by ysqysq123** [antvis/X6#3205] [Q&A]
node.attr('rect/title', 'Description of the rectangle')  鼠标悬浮到节点空白区域显示提示信息能起作用 
node.attr('text/title', 'Description of the rectangle')  鼠标悬浮到节点文字显示提示信息不起作用

**Comment by NewByVector** [comment]
![image](https://user-images.githubusercontent.com/20186737/217130304-e0a406e6-5c4c-42b4-a537-226de52a94cf.png)
我试了下 node.attr('text/title', 'Description of the rectangle') 是有效的。

**Comment by ysqysq123** [comment]
![b6b9b0ad86173ff5cc75b6c56a48451](https://user-images.githubusercontent.com/104961042/217131946-40c7b355-029c-4ebe-bc88-e7cbac4c3afd.jpg)
![b5780fd225afb9860f569a6811b1d33](https://user-images.githubusercontent.com/104961042/217131947-135147b6-c2a2-4165-9dfc-6ec97472113d.jpg)

**Reply by NewByVector** [reply]
有 demo 看下吗？

**Reply by ysqysq123** [reply]
https://github.com/ysqysq123/vue2-antv-x6.git
你好，是鼠标悬浮到文字上面无效（浏览器：chrome 版本：108.0.5359.125， 操作系统win10）
